src/dumpling-service/dumpling.web/App_Start/AuthConfig.cs

```diff
@@ -0,0 +1,83 @@
+﻿using DumplingLib;
+using Microsoft.Azure.KeyVault;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+using System.Web;
+using System.Web.Configuration;
+
+namespace dumplingWeb.App_Start
+{
+    public class AuthConfig
+    {
+
+        public static class CertificateHelper
+        {
+            public static X509Certificate2 FindCertificateByThumbprint(string findValue)
+            {
+                X509Store store = new X509Store(StoreName.My, StoreLocation.LocalMachine);
```

Comment:
it seems like the X509Store class is IDisposable (from MSDN: public sealed class X509Store : IDisposable, https://msdn.microsoft.com/en-us/library/system.security.cryptography.x509certificates.x509store(v=vs.110).aspx) 

so this should be better coded (more clearly) as 

using(X509Store store = new X509Store(StoreName.My, StoreLocation.LocalMachine))
{
/// ...
}